Fix naming convention lint warnings

When we bumped our `@typescript-eslint` version in https://github.com/cursorless-dev/cursorless/pull/1989, we started getting many lint warnings like the following:

```
/home/runner/work/cursorless/cursorless/packages/cheatsheet-local/src/webpack.config.ts
  8:8  warning  Import name `HtmlWebpackInlineSourcePlugin` must match one of the following formats: camelCase  @typescript-eslint/naming-convention
  9:8  warning  Import name `HtmlWebpackPlugin` must match one of the following formats: camelCase              @typescript-eslint/naming-convention
```

See eg https://github.com/cursorless-dev/cursorless/actions/runs/6840283981/job/18599382888. I'm not sure why we would be getting these warnings, as the [default config](https://typescript-eslint.io/rules/naming-convention/#options) looks like it should allow PascalCase for imports 🤔

--- a/package.json
+++ b/package.json
@@ -29,8 +29,8 @@
   "devDependencies": {
     "@pnpm/meta-updater": "1.0.0",
     "@types/node": "^18.18.2",
-    "@typescript-eslint/eslint-plugin": "^6.9.0",
-    "@typescript-eslint/parser": "^6.9.0",
+    "@typescript-eslint/eslint-plugin": "^6.10.0",
+    "@typescript-eslint/parser": "^6.10.0",
     "esbuild": "^0.17.11",
     "eslint": "^8.53.0",
     "eslint-config-prettier": "^8.8.0",

